services/gateway_manager/src/main/java/com/futurewei/alcor/gatewaymanager/dao/RouteTableRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.futurewei.alcor.gatewaymanager.dao;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class RouteTableRepository {
+}
```

Comment:
@songxiaoyan We probably don't need it.

Reply:
ok, I can deleted reserved interface if don't need